### PR TITLE
updated file

### PR DESCRIPTION
**Pull request by mkutter72**

### CODE REVIEW

File: lib/assessment.js [hunk 1]
```diff
@@ -5,30 +5,67 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
+  if (arguments.length > 0) {
 
+    var array = arguments['0'];
```
Comment (MattB-GA-Boston) [inline lib/assessment.js]: Should be `arguments[0]`, not `['0']`.

File: lib/assessment.js [hunk 2]
```diff
@@ -5,30 +5,67 @@
...
+    var retMin = array[0];
```
Comment (GA-MEB) [inline lib/assessment.js:11]: Although the previous line will work, this one will not (unless the first element passed into `min` is an array).

File: lib/assessment.js [hunk 3]
```diff
@@ -5,30 +5,67 @@
...
+    for (var i = 1; i < array.length ; ++i)
+      if (array[i] < retMin)
+        retMin = array[i];
+   return retMin;
+  };
```
Comment (GA-MEB) [inline lib/assessment.js:16]: Don't forget your curly braces!

File: lib/assessment.js [hunk 4]
```diff
@@ -40,6 +77,15 @@ var newArray2 = function newArray2(length, defaultsFunction) {
 // the object passed in as an argument or the object created
 var addMethod = function addMethod(propertyName, method, obj) {
 
+  if (obj === undefined)
+  {
+    obj = new {};
```
Comment (GA-MEB) [inline lib/assessment.js:82]: You could just write `obj = {};`

**Comment by GA-MEB** [comment]
3.5 of 5 (-0.5 on Q1, Q2; -0.25 on Q3, Q4)

You're missing lots of curly braces - watch out for that - but it's clear from your code that you've got the general idea of what to do. Keep practicing, and eventually that syntax will stick!

**Pull request closed without merging**